src/py/flwr/server/driver/grpc_driver.py

```diff
@@ -46,92 +46,94 @@
 ERROR_MESSAGE_DRIVER_NOT_CONNECTED = """
 [Driver] Error: Not connected.
 
-Call `connect()` on the `GrpcDriverHelper` instance before calling any of the other
-`GrpcDriverHelper` methods.
+Call `connect()` on the `GrpcDriverStub` instance before calling any of the other
+`GrpcDriverStub` methods.
 """
 
 
-class GrpcDriverHelper:
-    """`GrpcDriverHelper` provides access to the gRPC Driver API/service."""
+class GrpcDriverStub(DriverStub):
+    """`GrpcDriverStub` provides access to the gRPC Driver API/service."""
 
     def __init__(
         self,
         driver_service_address: str = DEFAULT_SERVER_ADDRESS_DRIVER,
         root_certificates: Optional[bytes] = None,
     ) -> None:
+        event(EventType.DRIVER_CONNECT)
```

Comment:
Constructors should not have side effects (for example, making network requests), so it's better to move this into a separate call that happens when the object is being used. This was one of the motivations behind the previous `connect` function.